pkg/tenant/api/v1/api.go

```diff
@@ -48,51 +44,44 @@ type StatusReply struct {
 // Tenant Requests and Responses
 //===========================================================================
 
+type PageQuery struct {
+	PageSize      uint32 `url:"page_size,omitempty"`
+	NextPageToken string `url:"next_page_token,omitempty"`
+}
+
 type Tenant struct {
 	ID              string `json:"id" uri:"id" binding:"required"`
 	TenantName      string `json:"tenant_name"`
 	EnvironmentType string `json:"environment_type"`
 }
 
-type PageQuery struct {
-	PageSize      uint32 `url:"page_size,omitempty"`
-	NextPageToken string `url:"next_page_token,omitempty"`
-}
-
 type TenantPage struct {
 	Tenants       []*Tenant
 	PrevPageToken string
 	NextPageToken string
 }
 
-type App struct {
-	ID      string `json:"id" uri:"id" binding:"required"`
-	AppName string `json:"app_name"`
-}
-
-type AppQuery struct {
-	Query         string
-	NextPageToken string
+type TenantMember struct {
+	TenantID   string `json:"tenant_id"`
```

Comment:
I wasn't sure if listing the members this way or passing in members from the `Member` struct would be best. Since this struct is tied specifically to the `tenant` I thought this would be better. If this is the best way to go then I'll need to update the `Tenant struct` to display `TenantID` instead of `ID`.